data/donations.js

```diff
@@ -0,0 +1,157 @@
+module.exports = {
+  allDonations() {
+    return [
+      {
+        _id: 1,
+        name: 'Center Table',
```

Comment:
This is dummy data for now, I'll update this code in my next ticket when I'll be working on the donations api.